configs/relengworker-prod/config.yml

```diff
@@ -117,3 +117,29 @@ worker_types:
         sla_seconds: 120
         capacity_ratio: 1.0
         min_replicas: 1
+
+  - worker_type: gecko-3-addon
+    provisioner: scriptworker-k8s
+    deployment_namespace: prod-addon
+    deployment_name: addon-prod-relengworker-firefox-1
+    autoscale:
+      algorithm: sla
+      args:
+        max_replicas: 10
+        avg_task_duration: 240
+        sla_seconds: 480
+        capacity_ratio: 1.0
+        min_replicas: 1
+
+  - worker_type: gecko-1-addon
+    provisioner: scriptworker-k8s
+    deployment_namespace: prod-addon
+    deployment_name: addon-prod-relengworker-fake-firefox-1 
+    autoscale:
+      algorithm: sla
+      args:
+        max_replicas: 10
```

Comment:
Should we tune this down a bit? I see in AWS we only had [two machines](https://tools.taskcluster.net/provisioners/scriptworker-prov-v1/worker-types/addon-dev) so maybe we should keep it the same in the new world? I'm thinking of the staging releases that we'll likely to run in the future. They are going to be faster which is good but also more expensive. Should we worry about that treshold now?

Reply:
We could do, I was following the other workers setting gecko-1 the same as gecko-3. If we want to find issues in staging it would make sense to minimize differences where possible. They'll sit on 1 replica most of the time, I hope.

Reply:
Okay, let's leave it like this for now and we can tune it down later on if we need to.